src/components/CommentCard/CommentCard.tsx

```diff
@@ -33,87 +34,78 @@ export interface CommentCardProps {
+                  </time>
+                </em>
+              </p>
+            )}
+            {comment.sentiment && (
+              <div>
+                <div className="govuk-heading-s">
+                  Sentiment towards this application
+                </div>
+                <p className="govuk-body">
+                  {capitaliseWord(comment.sentiment)}
+                </p>
+              </div>
             )}
-          </h3>
-          {comment?.receivedDate && (
-            <p className="govuk-body">
-              <em>
-                Published{" "}
-                <time dateTime={comment.receivedDate}>
-                  {formatDateTimeToDprDate(comment.receivedDate)}
-                </time>
-              </em>
-            </p>
-          )}
-          {comment.sentiment && (
-            <div>
-              <div className="govuk-heading-s">
-                Sentiment towards this application
+            <div className="govuk-heading-s">Comment</div>
+            <div className="comment-text">
+              <div className="govuk-body">
+                {summary}
+
+                {!isExpanded && continued && (
+                  <span className="read-more-ellipsis">...</span>
+                )}
+                {isExpanded && continued && <span>{continued}</span>}
               </div>
-              <p className="govuk-body">{capitaliseWord(comment.sentiment)}</p>
+
+              {continued && (
+                <button
+                  aria-expanded={isExpanded}
+                  aria-controls={`comment-${commentId}`}
+                  onClick={() => setIsExpanded(!isExpanded)}
+                  className="govuk-body govuk-link govuk-link--no-visited-state dpr-comment-card--toggle"
+                >
+                  {isExpanded
+                    ? "Minimise this comment"
+                    : "Read the rest of this comment"}
+                </button>
+              )}
             </div>
-          )}
-          <div className="govuk-heading-s">Comment</div>
-          <div className="comment-text">
-            <p className="govuk-body">{comment?.comment}</p>
           </div>
         </div>
-        {isOverflowing && !isExpanded && (
-          <div className="read-more-ellipsis">...</div>
-        )}
-        {isOverflowing && (
-          <button
-            className="govuk-body govuk-link govuk-link--no-visited-state dpr-comment-card--toggle"
-            onClick={toggleExpand}
-          >
-            {isExpanded
-              ? "Minimise this comment"
-              : "Read the rest of this comment"}
-          </button>
-        )}
       </div>
-    </div>
-  );
+    );
+  } else {
+    return (
+      <div className="comment-card">
+        <div className="comment-card__skeleton-item"></div>
+        <div className="comment-card__skeleton-item"></div>
+        <div className="comment-card__skeleton-item"></div>
+      </div>
```

Comment:
A skeleton view is designed to "block out" the content until the real data loads, so we get a seamless transition and no layout shift when the real content loads.

![image](https://github.com/user-attachments/assets/5ad315ca-3588-4eac-acd0-0a378b71fcda)


Can you update the skeleton_items to match the content they’ll be replaced by? Right now, there will be a noticeable jump when everything loads in!

<img width="744" alt="image" src="https://github.com/user-attachments/assets/319ba1d6-720c-4051-a307-c143943a2075" />


![image](https://github.com/user-attachments/assets/0852b260-71df-4b52-b6d5-862f35ea157b)